aips/aip-116.md

```diff
@@ -26,29 +26,25 @@ Authenticating with Aptos accounts is a privacy-first solution that empowers ind
 
 By authenticating using Aptos accounts, users have full control over their authentication journey. They can decide _what to share_ and _how to log in,_ ensuring a more secure and customizable experience.
```

Comment:
This paragraph and the one above it are fluff (they are not saying anything meaningful). I think you should remove them and just start with what you wrote below "The existing `connect` + `signMessage` flow"